dao/src/main/java/se/fortnox/reactivewizard/db/transactions/TransactionStatement.java

```diff
@@ -16,6 +15,11 @@ public TransactionStatement(Transaction transaction) {
         this.statement = new AtomicReference<>();
     }
 
+    public TransactionStatement(Transaction transaction, Statement statement) {
```

Comment:
Cannot find any usage of this constructor

Reply:
Oops, I was thinking about removing the add and markStatementSubscribed just as you mentioned in other comments.